lib/otx_ruby/ip.rb

```diff
@@ -101,5 +101,15 @@ def get_http_scans(ip, type = :ipv4)
 
       return results
     end
+
+    def nids_list(ip, type = :ipv4)
```

Comment:
Probably worth Documenting this function, especially that 'type' field and the options for it